load.php

```diff
@@ -29,6 +29,14 @@
 		'Core',
 		[
 			'enabled' => true,
+			'consent' => [
+				'display_banner'      => false,
+				'privacy_policy_page' => '',
+				'cookie_policy_page'  => '',
+				'cookie_expiration'   => 30,
+				'banner_options'      => 'none',
+				'banner_text'         => '',
```

Comment:
It's been a little inconsistent but the bulk of our config options use hyphens rather than underscores. I think that was never standardised in the config RFC so I should add an RFC for standardising these things.